### PR TITLE
Docs: Add the page of 1127 meetup

### PR DESCRIPTION
**Pull request by ThreeMonth03**
In this pull request, I update the page of 11/27 meetup.

### CODE REVIEW

File: content/pages/meetup/2024/1127-nycu.rst [hunk 1]
```diff
@@ -0,0 +1,153 @@
+========================================
+Meetup 2024 November 27th in NYCU
+========================================
+
+:date: 2024-11-20 20:59
+:url: meetup/2024/1127-nycu
+:save_as: meetup/2024/1127-nycu.html
+
```
Comment (ThreeMonth03) [inline content/pages/meetup/2024/1127-nycu.rst:8]: Modify the date.

File: content/pages/meetup/2024/1127-nycu.rst [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+you can come to the meetup and discuss the questions with the teaching assistant.
+
+|
+
+Sign up
+------------
+
+The meetup is free. 
+Please register on `discord event <https://discord.com/channels/730297880140578906/1007075707400237067/1308750422110109716>`__. 
+Click the green check mark to participate the meetup.
```
Comment (ThreeMonth03) [inline content/pages/meetup/2024/1127-nycu.rst:110]: Register link.

File: content/pages/meetup/index.rst [hunk 3]
```diff
@@ -9,6 +9,9 @@ meetup
 meetup 2024
 ==============
 
+* `Meetup 2024 November 27th at NYCU
+  <{filename}2024/1127-nycu.rst>`__
+
```
Comment (ThreeMonth03) [inline content/pages/meetup/index.rst:14]: The internal link directs to the page of 11/27 meetup.

**Review by ThreeMonth03 — COMMENTED** [review]
@terrychan999 @Vincent-Lien Could you please help review this pull request?

**Comment by terrychan999** [comment]
> @terrychan999 @Vincent-Lien Could you please help review this pull request?

@ThreeMonth03 Thanks, LTGM~

**Pull request merged**